README.md

```diff
@@ -106,14 +106,32 @@ Note that every `header` implementation must append all required blocks within i
 
 Through the Storefront, you can change the headers's behavior and interface. However, you also can make in your theme app, as Store theme does.
 
-| Prop name       | Type      | Description                                       | Default value |
-| --------------- | --------- | ------------------------------------------------- | ------------- |
-| `leanWhen`      | `String`  | Cases in which the menu is in lean mode           | 'a^'          |
-| `linkUrl`       | `String`  | Address opened when the user clicks the logo      | '/'           |
-| `logoUrl`       | `String`  | URL of the logo image                             | N/A           |
-| `logoTitle`     | `String`  | Alt text for the logo                             | N/A           |
-| `showSearchBar` | `Boolean` | Sets whether the search bar is visible or not     | true          |
-| `showLogin`     | `Boolean` | Sets whether the login button is displayed or not | true          |
+| Prop name              | Type      | Description                                       | Default value                                                            |
```

Comment:
I don't think it's "developer friendly" to have all these spaces between the columns on the markdown. Specially if you don't use an autoformatter for `.md` files, it makes really hard to edit these docs and maintain the visual alignment.

Reply:
done ✅